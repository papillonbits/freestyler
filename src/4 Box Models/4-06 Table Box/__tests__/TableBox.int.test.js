import { regular } from './TableBox.int.story'

describe('<TableBox />', () => {
  describe('Render', () => {
    test('must match regular()', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })
  })
})
