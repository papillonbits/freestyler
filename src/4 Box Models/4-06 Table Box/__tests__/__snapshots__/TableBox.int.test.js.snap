// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TableBox /> Render must match regular() 1`] = `
<div>
  <div
    class="table-box"
  >
    <h1>
      Table Box
    </h1>
    <div
      class="container"
    >
      <table
        class="default"
      >
        <tr>
          <td>
            BEFORE
          </td>
        </tr>
      </table>
      <table
        class="table"
      >
        <tr>
          <td
            class="cell"
          >
            Table Cell 
          </td>
          <td
            class="cell"
          >
            Table Cell
          </td>
        </tr>
      </table>
      <table
        class="default"
      >
        <tr>
          <td>
            AFTER
          </td>
        </tr>
      </table>
    </div>
  </div>
</div>
`;
